locustfile.py

```diff
@@ -1,12 +1,12 @@
 """
 This is a (much needed) load test for this repo. These are some example usages:
```

Comment:
Outside the scope of this PR, but why is it much needed?

Reply:
We had some performance problems in production, so to get a view of the actual performance we do the locust tests.